src/browser/provider/built-in/dedicated/chrome/browser-client.ts

```diff
@@ -161,12 +166,14 @@ export class BrowserClient {
     }
 
     public async getActiveClient (): Promise<remoteChrome.ProtocolApi> {
-        const client = this._clients[this._clientKey];
+        let client = this._clients[this._clientKey];
```

Comment:
Method `getActiveClient` returns `remoteChrome.ProtocolApi`  or `null`.
So, we should change the method return value instead of casting.

Reply:
the `getActiveClient` method can return `null`. So, I do not want to change its return value. However, since, this method returns `null` the TS checker forces me to add null check in all places. I would prefer to get an exception instead of adding the null check in possible places. We really need to check if the value is null only in one place. So, my problem is in TS checker. Let's discuss it verbally